SetupGuide_Plugin_Admin.php

```diff
@@ -627,7 +627,7 @@ public function config_objcache() {
 	 * @see \W3TC\Util_Http::get_headers()
 	 */
 	public function test_browsercache() {
-		if ( wp_verify_nonce( $_POST['_wpnonce'], 'w3tc_wizard' ) ) { // phpcs:ignore WordPress.Security.ValidatedSanitizedInput
+		if ( wp_verify_nonce( Util_Request::get_string( '_wpnonce' ), 'w3tc_wizard' ) ) { // phpcs:ignore WordPress.Security.ValidatedSanitizedInput
```

Comment:
Remove the phpcs comment to have it validate the change.

Reply:
Removed PHPCS disable comment.